website/blog/apr-2023/silk2170.md

```diff
@@ -0,0 +1,167 @@
+
+Direct2D is a 2D vector graphics application programming interface (API) that allows for the creation of visually rich 2D graphics. This marks our first binding to a native library dedicated solely to vector graphics. On the other hand, DirectComposition provides advanced compositing and visual effects capabilities to enable high-performance bitmap composition with transforms, effects, and animations. You can use our DirectComposition bindings to create visually engaging user interfaces that feature rich and fluid animated transitions from one visual to another. Finally, DirectWrite is a text layout and rendering API that supports high-quality text rendering for both display and print, quite closely integrated with Direct2D. Finally, WIC (Windows Imaging Component) is a framework for working with images and image metadata - this is again closely integrated with Direct2D.
+
+We're especially proud to note that much of the work on these new bindings was done by a community member, @curin, who took the initiative to drive these additions. The Silk.NET team only had to do minor rework to integrate them into the library. This is a great example of how Silk.NET is a true community-driven project, with users and contributors working together to create a better development experience for everyone.
+
+## Platform Expansion
+
+Development Lead: Dylan Perks (@Perksey) / Beyley Thomas (@Beyley)
+
+In Silk.NET 2.17, we focused heavily on expanding platform support and increasing the portability of Silk.NET code. To achieve this, we made Silk.NET compatible with reflection-free NativeAOT, previously known as CoreRT, which enables developers to create minimal distributions of their games and applications. Additionally, we added support for Android 12, which was previously incompatible with Silk.NET 2.16 due to recent changes in the Android permissions API.
+
+In addition to expanding platform support and portability, we have also been doing increased testing on Windows on ARM using the Volterra development kit. As a result, we're excited to announce that Silk.NET is now fully compatible with Windows on ARM. While your mileage with the Volterra graphics drivers may vary, we have added D3D11-on-12 bindings in 2.17 to allow developers to wield the Direct3D 11 API you all know and love using the Direct3D 12 drivers, which tend to be in better shape on newer devices. 
+
+But perhaps the biggest news is the newfound portability of Silk.NET across different operating systems. For developers who prefer DirectX over other graphics APIs, they can now use the Direct3D bindings they love almost out-of-the-box on Linux and macOS using the DXVK and VKD3D native libraries, which Silk.NET now has full support for. In fact, we've made one of our examples DXVK compatible, and it only took just a few lines of code changes to make it fully portable across all three major desktop operating systems using Silk.NET.Windowing.
```

Comment:
no MacOS on the DXVK or VKD3D side right now, VKD3D would probably be quite shrimple due to the zig toolchain for the shader compiler, but DXVK would be hard to do on our end alone, so we'll depend on upstream to tackle that (its been brought up before there)

Reply:
Removed

Reply:
the final sentence was not updated, but im fixing that myself in a small commit, along with some small clarifications for DXVK/VKD3D, given the not unreasonable assumption that using VKD3D means D3D12 support, even though we are locked to D3D9/D3D11 with DXVK